pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -82,9 +83,13 @@ func RunClusterInfo(factory *cmdutil.Factory, out io.Writer, cmd *cobra.Command)
 					link += "http://" + ip + ":" + strconv.Itoa(port.Port) + " "
 				}
 			} else {
-				link = client.Host + "/api/v1beta3/proxy/namespaces/" + service.ObjectMeta.Namespace + "/services/" + service.ObjectMeta.Name
+				link = client.Host + "/api/" + latest.Version + "/proxy/namespaces/" + service.ObjectMeta.Namespace + "/services/" + service.ObjectMeta.Name
```

Comment:
Hi @piosz @saad-ali, you could merge this one without #9692. There is more work need to be done in #9692. I will rebase later. Thank you.